src/transaction-flow/input/ProfileEditor/ProfileEditor-flow.tsx

```diff
@@ -192,6 +198,14 @@ const ProfileEditor = ({ data = {}, transactions = [], dispatch, onDismiss }: Pr
     }
   }, [status, statusLoading, resumable])
 
+  const { isWrapped } = useNameDetails(name)
```

Comment:
since you don't need all of the data from `useNameDetails()` would be better to just take the calculation for `isWrapped` out, make it a util func, and make a new hook called `useOwner()` or similar

Reply:
isn't useQuery caching anything heavy